results/cstr_results_25_dantzig.csv

```diff
@@ -0,0 +1,630 @@
+Method,Approach,Solver,Objective,Time,Status,User_time,NT
```

Comment:
What't the point of adding this file if the 30 is added as well? Unify them

Reply:
Those are the ones that I did before I did for 25 CSTRs. For the 30 CSTRs, I did run the code separately. Shall I remove the 25 and let only the 30 CSTR results remain?

Reply:
Yes. No need to have the puplicate. You can save a version locally just in case but keep the repository clean